ebu_tt_live/documents/test/test_converters.py

```diff
@@ -47,9 +50,131 @@ def test_simple(self):
 
     def test_ericsson_1(self):
```

Comment:
Does the test name need to change now that you're using a different file name?

Reply:
sigh, I suppose it should